.github/scripts/eval_run.py

```
@@ -0,0 +1,38 @@
+import requests
+import argparse
+import sys
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)